tools/docker/syzbot/Dockerfile

```diff
@@ -29,25 +29,20 @@ RUN test "$(uname -m)" != x86_64 && exit 0 || \
 RUN curl https://dl.google.com/go/go1.22.7.linux-$(uname -m | sed 's/aarch64/arm64/;s/x86_64/amd64/').tar.gz | tar -C /usr/local -xz
 ENV PATH /usr/local/go/bin:$PATH
 
-# The default clang-14 is too old, install the latest one.
 RUN apt-get install -y -q gnupg software-properties-common apt-transport-https
+# The default clang-14 is too old, install the latest one.
+# See https://kernel.org/pub/tools/llvm.
+ARG LLVM_VERSION=llvm-15.0.7-x86_64
+COPY provision-llvm.sh /tmp/provision-llvm.sh
+RUN /tmp/provision-llvm.sh $LLVM_VERSION
+
 RUN curl https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -
 RUN add-apt-repository "deb http://apt.llvm.org/bookworm/ llvm-toolchain-bookworm-15 main"
 RUN apt-get update --allow-releaseinfo-change
-RUN apt-get install -y -q --no-install-recommends llvm-15 clang-15 clang-format-15 clang-tidy-15 lld-15
+RUN apt-get install -y -q --no-install-recommends clang-format-15 clang-tidy-15
```

Comment:
It is not a blocker, but -> https://docs.docker.com/build/building/best-practices/#apt-get